src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp

```diff
@@ -239,6 +239,29 @@ class OVCompiledModelClassBaseTestP : public OVClassNetworkTest,
     }
 };
 
+using GetMetricExecutionDeviceTestParams = std::tuple<std::string,
+                                                      std::string
+                                                     >;
+
+class OVClassExecutableNetworkGetMetricTest_ExecutionDevice : public OVClassNetworkTest,
```

Comment:
src/tests/functional/plugin/shared/include/behavior/ov_plugin/properties_tests.hpp

there is a OVCompileModelGetExecutionDeviceTests in place, is it able to re-use?

Reply:
Reused this class, thanks.